tests/pull-update/test.bats

```diff
@@ -34,7 +50,8 @@ teardown_file() {
 }
 
 @test "Pull and update second version policy" {
-	run $CONFTEST test --policy "${TEMP_DIR}/policy" --update "file://${TEMP_DIR}/remote-policy/b" file.json
+	# Use git:: prefix for local git repository to avoid file:// symlink issues on Windows
```

Comment:
Please revert this. The test should be able to run without forcing the Git protocol.